<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalExamDialog</class>
 <widget class="QWidget" name="FinalExamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>682</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="10" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnNext">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Click here to go to the next question</string>
            </property>
            <property name="text">
             <string>&amp;Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMode">
            <property name="text">
             <string>Test Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDone">
            <property name="toolTip">
             <string>Im Finished Memorizing Scriptures!</string>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdMins"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdSecs">
            <property name="numDigits">
             <number>2</number>
            </property>
            <property name="value" stdset="0">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="txtQuestionNbr">
          <property name="toolTip">
           <string>Scripture Chapter Number</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="toolTip">
           <string>Question:</string>
          </property>
          <property name="text">
           <string>Question:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label194">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>Course</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_212">
          <property name="toolTip">
           <string>Scripture Chapter</string>
          </property>
          <property name="text">
           <string>Question No.:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtQuestionNbr</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>BiblePay - Final Exam Certification Testing Area     &lt;small&gt;  v1.0&lt;/small&gt;</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_31">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Your Answer:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_2242">
          <property name="toolTip">
           <string>Scratchpad</string>
          </property>
          <property name="text">
           <string>Comments/Answer:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="txtAnswer">
          <property name="toolTip">
           <string>Scratchpad</string>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="txtQuestion">
          <property name="toolTip">
           <string>Exam Question</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>128</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>256</height>
           </size>
          </property>
          <property name="title">
           <string>Your Answer:</string>
          </property>
          <widget class="QRadioButton" name="radioA">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>777</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>777</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>A</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioB">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>777</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>777</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>B</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioC">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>777</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>777</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>C</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioD">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>777</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>777</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>D</string>
           </property>
          </widget>
          <widget class="QLabel" name="lblGrade">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>201</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string> </string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QPushButton" name="btnBack">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>* This is the Final Exam for BiblePay University, College Course Material. &lt;br&gt;* In TEST mode, you will be tested and graded on the material.  In REVIEW mode, you will be able to see the answers.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="txtInfo">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtQuestionNbr</tabstop>
  <tabstop>btnNext</tabstop>
 </tabstops>
 <resources>
  <include location="../biblepay.qrc"/>
 </resources>
 <connections/>
</ui>
